app/src/main/res/layout/offer_header.xml

```diff
@@ -73,18 +73,23 @@
 
             <androidx.constraintlayout.widget.ConstraintLayout
                 android:id="@+id/startDateContainer"
+                style="?materialButtonTextStyle"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:layout_margin="@dimen/base_margin_double"
-                android:background="@drawable/background_pick_date"
+                android:background="@drawable/background_rounded_corners_ripple"
```

Comment:
Given that we are on API 23, we can use `android:foreground="?selectableItemBackground"` 😉

Reply:
I want rounded corners though, with foreground the whole layout is targeted (which is squared)

Reply:
Sensible!

Reply:
`?selectableItemBackgroundBorderless` is rounded though, although it's not a squircle, to be fair. You can check it out if you want, but it's not super-important.